### DISCUSSION TITLE
Simplification of setup - Questions

### DISCUSSION THREAD

**Discussion by NickHowes** [probonopd/MiniDexed#200] [Q&A]
Looking to make this a bit simpler. Does anyone know how to modify the ini file permitting support (as Zynthian does) for the behringer UCA series of USB audio interfaces?  Also the 
DFRobot IIC 16x2 RGB LCD KeyPad HAT for Raspberry Pi V1.0, has controls on board and appears to be a vastly more convenient solution, as it's a hat... does this work/anyone know how to make it work if it doesn't . 

In essence, trying to get all of this running on a PI3, using an M-Audio 1x1 midi interface (which should just work), that LCD and the Behringer for audio (as it also has optical Toslink output, which will mean I can very cleanly stream to my mixing desk)

Many thanks

**Comment by craigyjp** [comment]
Wow, so nothing on the supported device list bit you want it all to work
for your instance only. That's a lot of development.

I think the current setup is perfectly adequate for the hardware it
supports, the software is being developed to offer more functionality for
everyone based on popularity.

**Comment by NickHowes** [comment]
Sorry if my questions warranted that response, I'll take my leave here, and work around it

**Reply by probonopd** [reply]
Hello @NickHowes. Most of us don't have this hardware. Unfortunately, USB audio is not supported at this time.

**Comment by diyelectromusic** [comment]
Re: Behringer: I'm not sure that is quite the use-case for the Dexed?  Isn't this an interface for a DAW whereas the MiniDexed is more of an instrument that might plug into a DAW via your Behringer?

Aside: Zynthian has a full instance of Linux underneath it so it can act more like a DAW "from the inside", but that is a lot of work for a "bare metal" environment like MiniDexed.  I think (full) Dexed is one of the available synth engines for Zynthian though if you want to go that route...

Re: DFRobot HAT: This uses the I2C display interface which isn't supported yet.  I think there are plans to consider it, so if it is, then this might "just work".  With regards to the input buttons, the key input method for MiniDexed is a rotary encoder which this board doesn't have.  Button support has been put forward as an idea, but again isn't there yet, but again if that does gets implemented at some point it might work with some configuration.  But if so that is in the future still.

Kevin
